app/main.ipynb

```diff
@@ -166,15 +248,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
-   "outputs": [],
+   "execution_count": 483,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "4535\n"
+     ]
+    }
+   ],
    "source": [
-    "# write your code for task 5 here"
+    "number_of_stupid_but_rich = dataset[(dataset['education'].isin(['Bachelors', 'Prof-school', 'Assoc-acdm', 'Assoc-voc', 'Masters', 'Doctorate'])) & (dataset['salary'] == '>50K')]\n",
+    "print(len(number_of_stupid_but_rich))"
```

Comment:
read the task again
Check, if there are some people **without** higher education (education: Bachelors, Prof-school, Assoc-acdm, Assoc-voc, Masters, Doctorate), but with > 50K salary"